### PR TITLE
home page and logistration pages changes

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**
# Blog App Frontend:
Description: Home Page and Logistration pages are now changed as per the requirements. Most importantly, a Sticky Footer and Default Image has been added.
Installation Instructions:

1. run `npm install`
2. run npm start

Screenshot from 2019-10-01 11-17-06
Screenshot from 2019-10-01 11-17-30
Screenshot from 2019-10-01 11-18-36
Screenshot from 2019-10-01 11-18-49
Screenshot from 2019-10-01 11-45-18

### CODE REVIEW

File: src/components/accounts/Register.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ export class Register extends Component {
                     <h2 className="text-center">Register</h2>
                     <form onSubmit={this.onSubmit}>
                         <div className="form-group">
-                            <label>Username</label>
+                            <label>Username*</label>
```
Comment (zubair-arbi) [inline src/components/accounts/Register.js]: make these required fields accessible.

**Review by zubair-arbi — CHANGES_REQUESTED** [review]

File: src/components/accounts/Register.js (hunk 1 above)
Comment (Ali-D-Akbar) [reply src/components/accounts/Register.js]: Done.

**Review by zubair-arbi — APPROVED** [review]
👍

**Comment by zubair-arbi** [comment]
Remember to squash the commits before merge.

**Pull request merged**